apps/topics/models/taggable_model.py

```diff
@@ -56,14 +54,14 @@ def tags_string(self) -> str:
         return ''
 
     @property
-    def tags_html(self) -> SafeString:
+    def tags_html(self) -> str:
         """Return the model instance's tags as an HTML string of <li> elements."""
         tags_html = ''
         if self.tag_keys:
             tags_html = ' '.join(
                 [f'<li class="topic-tag"><a>{tag_key}</a></li>' for tag_key in self.tag_keys]
             )
-        return format_html(tags_html)
+        return tags_html
```

Comment:
Is formatting not needed anymore or maybe done somewhere else now?

Reply:
It's not needed anymore, since we are no longer using Django templates for pages that display topic tags. (`format_html` is just a helper for escaping HTML that's rendered in Django templates.)